### PR TITLE
Build the root stylesheet (CSS resets, color schemes) with webpack

### PR DESCRIPTION
**Pull request by jsnajdr**
After all components are migrated, let's build the root stylesheet with webpack, too, and keep the `node-sass` build step only for `editor.css` and `directly.css` styles (used by TinyMCE and Directly iframes/widgets).

The root stylesheet is not owned by any particular component and contains:
- CSS reset styles for HTML elements
- Calypso styles for HTML elements like `a` links, `input` and `select` form elements
- color themes definitions

Steps:
- rename the `assets/stylesheets` directory to `client/assets/stylesheets`, so that we can `import 'assets/stylesheets/style.scss'` from JS sources and webpack resolver configuration finds them
- import the `assets/stylesheets/style.scss` in all webpack entrypoints
- load the `entry-main` CSS chunk in specialized pages like 404 and 500 error pages and in the `browsehappy` page. That replaces inclusion of the legacy `style.css` link
- remove code that adds the `style.css` links and switches them on LTR/RTL locale change. All CSS is now in webpack chunks and managed by the webpack runtime.

**How to test:**
- test regular Calypso page styles
- test 404 and 500 HTTP error pages
- test Desktop app
- test that Classic Editor TinyMCE is still styled well
- test that the Directly widget is still styled well

### CODE REVIEW

File: server/pages/index.js [hunk 1]
```diff
@@ -53,14 +52,7 @@ const debug = debugFactory( 'calypso:pages' );
...
 
-const staticFiles = [
-	{ path: 'style.css' },
-	{ path: 'editor.css' },
-	{ path: 'tinymce/skins/wordpress/wp-content.css' },
-	{ path: 'style-debug.css' },
-	{ path: 'style-rtl.css' },
-	{ path: 'style-debug-rtl.css' },
-];
+const staticFiles = [ { path: 'editor.css' }, { path: 'tinymce/skins/wordpress/wp-content.css' } ];
```
Comment (sgomes) [inline server/pages/index.js]: 🎉

**Review by sgomes — APPROVED** [review]
Excellent work, @jsnajdr! This is a really nice step forward, and the culmination of a lot of work over a long time 👍 

I couldn't find anything obviously wrong after playing around with the live branch, but I'm not convinced my testing was thorough. I didn't test the desktop app, for example.

File: apps/notifications/src/panel/boot/stylesheets/style.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import '../../../../../../assets/stylesheets/shared/typography.scss';
+@import '../../../../../../client/assets/stylesheets/shared/typography.scss';
```
[the remark this answers is not in the document]
Comment (jsnajdr) [reply apps/notifications/src/panel/boot/stylesheets/style.scss]: This is the `notifications` app, that lives outside the Calypso proper. There's certainly a big opportunity to clean it up.

**Comment by jsnajdr** [comment]
Thanks @sgomes for reviewing! I just tested the Desktop app and it works :+1: Let's merge.

**Pull request merged**